src/__tests__/requests.js

```diff
@@ -0,0 +1,25 @@
+import { should, use, request } from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../index';
+
+should();
+use(chaiHttp);
+
+describe('GET a user travel requests endpoint', () => {
+  it('return error if invalid parameter', (done) => {
+    request(server)
+      .get('/api/v1/requests/1a')
```

Comment:
Consider using template literals for url path that is reused across the code

Reply:
okay, thanks, will do